.github/workflows/cpp.yml

```diff
@@ -193,6 +193,11 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Install CMake 3.25.0
+        shell: bash
+        run: |
+          ci/scripts/install_cmake.sh 3.25.0 /tmp/local/
+          echo "/tmp/local/bin" >> $GITHUB_PATH
```

Comment:
We can avoid this by using `ubuntu-24.04`. (FYI: `ubuntu-latest` will use `ubuntu-24.04` soon.)